src/System.IO.Pipelines/pkg/System.IO.Pipelines.pkgproj

```diff
@@ -1,6 +1,9 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup>
```

Comment:
@wtgodbe @weshaggard Should this be removed in the merge?

Reply:
Yes we should remove this otherwise it will conflict with https://github.com/dotnet/corefx/pull/32203.